### DISCUSSION TITLE
please make an linux Appimage

### DISCUSSION THREAD

**Discussion by Nassi88** [simple64/simple64#461] [Ideas]
please make an appimage to make it portable and work for all linux distros

**Comment by loganmc10** [comment]
This is not going to happen. Flatpak is available on pretty much any Linux distro, I won't be maintaining multiple different packaging systems for this application.

**Comment by Nassi88** [comment]
Yes but flatpacks are not portable to use. I have an extern 4tb ssd and can take this to every pc to play roms. With appimage can a make an folder appimage.home an everything is saved there. I think appimage is much better solution. Much other Emulators is Going to appimage

**Comment by Nassi88** [comment]
If it's too much work for you, you could perhaps make an appimage for each major stable release and flatpack for the nightlys
